src/xrc/xmlres.cpp

```diff
@@ -2182,29 +2302,26 @@ void XRCConvertFromDLU(wxWindow* w, T& value)
     value = w->ConvertDialogToPixels(value);
 }
 
-// Helper for parsing values (of type T, for which XRCConvertFromAbsValue() and
-// XRCConvertFromDLU() functions must be defined) which can be expressed either
-// in pixels or dialog units.
 template <typename T>
 T
-ParseValueInPixels(wxXmlResourceHandlerImpl* impl,
+ParseStringInPixels(wxXmlResourceHandlerImpl* impl,
                    const wxString& param,
+                   const wxString& str,
```

Comment:
Pretty minor, but why rename this to `str` from `s`? The name is not that much better, but results in several diff hunks below, that would be avoided if we kept the old name.